charts/mapCharts/ChoroplethMap.tsx

```diff
@@ -277,9 +277,9 @@ export class ChoroplethMap extends React.Component<ChoroplethMapProps> {
             return { feature: d, distance: Vector2.distance(d.center, mouse) }
         })
 
-        const feature = sortBy(featuresWithDistance, d => d.distance)[0]
+        const feature = minBy(featuresWithDistance, d => d.distance)
 
-        if (feature.distance < 20) {
+        if (feature && feature.distance < 20) {
```

Comment:
Looking at the Lodash type sig of minBy I see that the type says it returns undefined, but in these cases that doesn't seem possible, right? So maybe `feature!` throughout? Not sure.

Reply:
I tend to go for the safer side in most cases, which involves explicitly checking for undefined even though it shouldn't happen, at least if it's not too much effort.
That way if we ever change some code path and a chart gets rendered for an empty dataset we are not faced with an error.

Also, I haven't looked thoroughly at the code that was there before, and even then there could've been some mistakes in there - `sortBy()[0]` can return undefined just as well as `minBy` can, it's just that TS doesn't catch such errors (that's also why we're using `_.first` in many cases).

So, if I have the (easy) choice I prefer `feature?.distance` over `feature!.distance`.